src/main/java/GameFacade.java

```diff
@@ -0,0 +1,56 @@
+import action.ActionManager;
+import geometry.Rectangle;
+import interaction.InteractionProcessor;
+import interaction.InteractionRuleBase;
+import interaction.InteractionType;
+import interaction.rule.BonusTankInteractionRule;
+import interaction.rule.TankBulletInteractionRule;
+import object.BonusHolder;
+import object.Bullet;
+import object.Tank;
+import object.Type;
+import object.manager.ObjectManager;
+import physic.PhysicManager;
+
+class GameFacade {
+
+    private ActionManager actionManager;
+    private PhysicManager physicManager;
+    private InteractionProcessor interactionProcessor;
+
+    void createGame(Rectangle fieldRectangle) {
+        this.actionManager = new ActionManager();
+        this.physicManager = new PhysicManager(fieldRectangle);
+
+        GameObjectInitializer gameObjectInitializer = new GameObjectInitializer();
```

Comment:
gameObjectInitializer is created here but used after objectManagers instantiaions